README.md

```diff
@@ -3,3 +3,12 @@
 Durst is a new notification daemon to act as the replacement of [dunst](https://github.com/dunst-project/dunst) on wayland based systems.
 
 It's written from scratch in Rust.
+
+## How to build and run durst
+```
+git clone https://github.com/durst-notification/durst.git
+cd durst
+mkdir -p "$HOME/.config/durst"
+cp config.yml "$HOME/.config/durst/config.yml"
```

Comment:
Ouch, this shouldn't be necessary. Having no config file at all shouldn't panic the program.

Reply:
Well it does. It fails [here](https://github.com/durst-notification/durst/blob/master/durst/src/config.rs#L33) with the message
```
thread 'main' panicked at 'Cannot parse YAML configuration file: EndOfStream', durst/src/config.rs:33:38
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

The thing is, what should the program do if the default configuration file actually contains some stuff? Do the default have to be both defined in the config file and in the program? This clearly hasn't gone well in dunst, since the default dunstrc isn't nearly the same as the defaults in code.
You could solve it by adding a defaults check in your CI, but it would still be a hassle. A better way would be to automatically create the default config from the defaults in code. It would take some more effort, but it would be worth it IMO.

Reply:
i agree with @bebehei . It should start without any config.
You are correct, that there is currently a problem.

generating a config would be nice, but I do not like it very much.
I prefer to override (predefined) options than to have to define them all individually. By generating configs, the user might think, that he is not allowed to remove options from the file. This would produce long config files in all dotfiles-repos, which i would like to prevent.

Reply:
Ah yes, that's a good point. You're right, it's better to have a shorter config. Then we have to make sure the CI can check if the config file doesn't differ from the defaults in code.

Reply:
Well yes, this should be fixed. I'm sorry for merging bad code here.

But it's better to document the current status quo instead of the state to be.

I'll fix these bits afterwards and make the `config.yml` optional (and adapt the docs then with this PR).